### ISSUE TITLE
Bug: Spaces in searches shows Forbidden error

### ISSUE THREAD

**Issue by ahmedxuae91** [MISP/MISP#9020]
### Expected behavior

Allow using spaces while searching events, tags or clusters.

Example: 
Eventinfo : private raas

### Actual behavior

While using spaces a 403 forbidden error is showing.

### Steps to reproduce

Search for any 2 words with space between them. This can be done anywhere in GUI searches.

![image](https://user-images.githubusercontent.com/21212837/232067261-43edffc0-96cf-47e0-8b23-7926527d74ce.png)

### Version

2.4.167

### Operating System

Ubuntu

### Operating System version

20.04

### PHP version

7.4

### Browser

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by vincenzocaputo** [comment]
It seems that the problem lies in the encoding of the space character in the URL. If you replace the space with the + character in the URL parameter (in your example: `/searcheventinfo:1stTerm+2ndTerm`), it will work.

I think it may be needed to edit the URI encoding in the JavaScript code (probably in the `app/webroot/js/misp.js` file) to fix this.

**Comment by ahmedxuae91** [comment]
> It seems that the problem lies in the encoding of the space character in the URL. If you replace the space with the + character in the URL parameter (in your example: `/searcheventinfo:1stTerm+2ndTerm`), it will work.
> 
> I think it may be needed to edit the URI encoding in the JavaScript code (probably in the `app/webroot/js/misp.js` file) to fix this.

Thanks @vincenzocaputo, Yes this is working well.
However, allowing spaces will be more convenient as we can add + in simple searches but it will be challenging in the advanced filters.
I would like to request MISP team to create another ticket to track the bug and close this one.

**Comment by Nicolas-Pellletier** [comment]
Yes i've got the same behaviour with space in search... I've open an [issue](https://github.com/MISP/MISP/issues/8996) lately for that. Thanks @vincenzocaputo for the way to fix this (I didn't try yet but i'm going to do it soon)
